### ISSUE TITLE
Bug: Response wont show

### ISSUE THREAD

**Issue by Doehl** [jarrodek/ChromeRestClient#1081]
Your description here

## Expected outcome
The response from the server should display.

## Actual outcome
An error in the bottom left is shown `Highlighter is unable to highlight the syntax`, for then to not show any data.

If i look in the console, the following error is shown:
```
Variables evaluator Function not found Rgb

HTMLElement.log (chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/index.js:102704:22)
    at HTMLElement._handleLog (chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/index.js:102690:16)
    at HTMLDocument.handler (chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/index.js:6026:17)
    at HTMLElement.fire (chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/index.js:6813:6)
    at HTMLElement._callFn (chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/index.js:67300:10)
    at HTMLElement._evalFunctions (chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/index.js:67275:25)
    at HTMLElement.<anonymous> (chrome-extension://hgmloofddffdnphfgcellkdfbfbjeloo/index.js:67129:22)
    at <anonymous>
```

## Versions
App: 10.0.3
Platform: 5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36

## Steps to reproduce
The response is a large json string.

**Comment by jarrodek** [comment]
Hi,

Thank you for the issue report.
The variables evaluator error is not related to highlighting and displaying the response data. 

I have to check how the body response display element works with invalid JSON data.

**Comment by jarrodek** [comment]
I've checked what is happening. 
So in case of very large responses then the syntax highlighter won't finish work in 5 seconds it informs the application about difficulties. The app terminates the thread that is trying to parse the data and opens the "raw" view which basically just displays unparsed data.

The only possibility is that the payload is so large that even Chrome has difficulties with displaying it. In my test scenario I waited about 20 seconds before anything showed up.
There's nothing I can do about it. It's how the browser renders the view.

I'm closing this as there's nothing I can do about it.
